src/components/BmiForm/BmiForm.jsx

```diff
@@ -1,33 +1,32 @@
 import React, { useState } from 'react';
-import '../App/App.css';
 import PropTypes from 'prop-types';
+import '../App/App.css';
```

Comment:
This is good, CSS imports should be at the bottom. I should've corrected this mistake of mine 🙄